mitosheet/src/components/modals/ReplayAnalysisPermissionsModal.tsx

```diff
@@ -0,0 +1,149 @@
+// Copyright (c) Mito
+
+import React, { useEffect, useState } from 'react';
+import MitoAPI from '../../jupyter/api';
+import { SavedAnalysis, UIState } from '../../types';
+import DefaultModal from '../DefaultModal';
+import TextButton from '../elements/TextButton';
+import { ModalEnum } from './modals';
+import { overwriteAnalysisToReplayToMitosheetCall } from '../../jupyter/jupyterUtils';
+import { isMitoError } from '../../utils/errors';
+import { TaskpaneType } from '../taskpanes/taskpanes';
+
+
+/**
+ * This function returns a summary view of the analysis. Note that we don't do anything
+ * super fancy here, as this will likely be a very rare flow as users will most often
+ * trust where they are getting their notebooks from.
```

Comment:
I think its worth doing the cleanup that I proposed. I think polish and around security practices is really important for user trust.